src/ukify/ukify.py

```diff
@@ -637,15 +645,23 @@ def pe_add_sections(uki: UKI, output: str):
         padp = section.PointerToRawData - oldp
         padsz = section.SizeOfRawData - oldsz
 
-        for later_section in pe.sections[i+1:]:
+        for later_section in pe.sections[i + 1 :]:
```

Comment:
The whitespace before `:` is unexpected, but it'd probably not worth fighting ruff over this…

Reply:
Yes, this is silly. My understanding is that it follows from the "space around binary operators rule" in PEP8.